python/sqlflow_submitter/tensorflow/predict.py

```diff
@@ -50,35 +50,15 @@ def define_tf_flags():
 class FastPredict:
     def __init__(self, estimator, input_fn):
         self.estimator = estimator
-        self.first_run = True
-        self.closed = False
         self.input_fn = input_fn
 
-    def _create_generator(self):
```

Comment:
@typhoonzero I don't understand why we need to use `self.next_feature` to access features. Please be aware that TF launches `_create_generator` in a separate thread, while `self.next_feature` introduces a sharing state, which is error-prone.